src/ppx_deriving_qcheck/README.md

```diff
@@ -20,12 +20,12 @@ let test =
 
 For `type tree` we derive two generators:
 - `val gen_tree : tree Gen.t` and
-- `val gen_tree_sized : ...`
+- `val gen_tree_sized : int -> tree Gen.t`
```

Comment:
Very minor - and my mistake: Can you add a newline here?
Otherwise line 24 is rendered as part of the second list item.
Sorry, I'm still more well-versed in LaTeX than Markdown 😅

Reply:
Here the fixup on fixup on commit comes !